samples/tab-stage-view/csharp/README.md

```diff
@@ -94,23 +94,7 @@ For reference please check [Tabs link unfurling and Stage View](https://docs.mic
 
 - Welcome message with feature explanation.
 
-![Welcome Message](TabInStageView/Images/welcomeAction.png)
```

Comment:
Check if you have linked your sample with the root readme file.

Reply:
Check if we have added this sample in build pipeline.

![image](https://user-images.githubusercontent.com/59052547/192456249-e0b39ff2-26d4-4cd6-b9c3-b209b713f081.png)

Reply:
Already added.